neurow/lib/neurow/internal_api/endpoint.ex

```diff
@@ -53,7 +56,15 @@ defmodule Neurow.InternalApi do
 
   get "/history/:topic" do
     history = Neurow.ReceiverShardManager.get_history(topic)
-    history = Enum.map(history, fn {_, {id, message}} -> %{id: id, message: message} end)
+
+    history =
+      Enum.map(history, fn {_, message} ->
```

Comment:
@achouippe Any better solution to serialize?

Reply:
@bpaquet , I replaced with

```
    history =
      Enum.map(history, fn {_, message} ->
        Map.from_struct(message)
      end)
```